webpack/assets/javascripts/bundle_multiselect.js

```diff
@@ -0,0 +1,2 @@
+require('multiselect');
```

Comment:
Sure, as long as you're sure that hound is using the repository rules and not the default. We can pick it up in PR code review if anyone does this, which I hope is not likely.